assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceTile/AudienceTileError.js

```diff
@@ -0,0 +1,160 @@
+	const hasInsufficientPermissionsError = errors.some( ( err ) =>
+		isInsufficientPermissionsError( err )
+	);
+
+	const requestAccessURL = useSelect( ( select ) =>
+		typeof select( MODULES_ANALYTICS_4 )?.getServiceEntityAccessURL ===
+		'function'
+			? select( MODULES_ANALYTICS_4 ).getServiceEntityAccessURL()
+			: null
+	);
+
+	const errorsWithSelectorData = useSelect( ( select ) =>
+		errors.map( ( err ) => {
+			const selectorData =
+				select( MODULES_ANALYTICS_4 )?.getSelectorDataForError( err );
+
+			return {
+				...err,
+				selectorData,
+			};
+		} )
+	);
+
+	const retryableErrors = errorsWithSelectorData?.filter(
+		( err ) =>
+			isErrorRetryable( err, err.selectorData ) &&
+			err.selectorData.name === 'getReport'
+	);
+
+	const dispatch = useDispatch();
+
+	const handleRetry = useCallback( () => {
+		retryableErrors.forEach( ( err ) => {
+			const { selectorData } = err;
+			dispatch( selectorData.storeName ).invalidateResolution(
+				selectorData.name,
+				selectorData.args
+			);
+		} );
+	}, [ dispatch, retryableErrors ] );
+
+	return (
+		<div className="googlesitekit-audience-segmentation-tile-error">
+			<div className="googlesitekit-audience-segmentation-tile-error__container">
+				<AudienceTileErrorImage className="googlesitekit-audience-segmentation-tile-error__image" />
+				<div className="googlesitekit-audience-segmentation-tile-error__body">
+					<div className="googlesitekit-audience-segmentation-tile-error__message">
+						<h3 className="googlesitekit-audience-segmentation-tile-error__title">
+							{ hasInsufficientPermissionsError
+								? __(
+										'Insufficient permissions',
+										'google-site-kit'
+								  )
+								: __(
+										'Data loading failed',
+										'google-site-kit'
+								  ) }
+						</h3>
+						{ hasInsufficientPermissionsError && (
+							<p className="googlesitekit-audience-segmentation-tile-error__explanation">
+								{ __(
+									"You'll need to contact your administrator.",
+									'google-site-kit'
+								) }
+								<br />
+								{ createInterpolateElement(
+									__(
+										'Trouble getting access? <HelpLink />',
+										'google-site-kit'
+									),
+									{
+										HelpLink: (
+											<Link
+												href="https://sitekit.withgoogle.com/documentation/troubleshooting/analytics/#insufficient-permissions"
+												external
+												hideExternalIndicator
+											>
+												{ __(
+													'Get help',
+													'google-site-kit'
+												) }
+											</Link>
+										),
+									}
+								) }
+							</p>
+						) }
+					</div>
+					<div className="googlesitekit-audience-segmentation-tile-error__cta">
+						{ hasInsufficientPermissionsError ? (
+							<Fragment>
+								<Button
+									href={ requestAccessURL }
+									target="_blank"
+									danger
+								>
+									{ __(
+										'Request access',
+										'google-site-kit'
+									) }
+								</Button>
```

Comment:
Did you consider reusing the `ReportErrorActions` component? Please check the IB of [8230](https://github.com/google/site-kit-wp/issues/8230).

cc: @techanvil

Reply:
Thanks @hussain-t. As we have that specced for #8230 and the changes and reuse of `ReportErrorActions` would be applicable here too, and would mean we can avoid copying part of `ReportErrorActions` here, that would be a good shout.